epitome/test/models_test.py

```diff
@@ -79,7 +81,7 @@ def test_train_early_stop_model(self):
 		assert m > 0.4 and m < 0.6
 
 		n_steps = 300
-		_, num_steps, _ = model.train(n_steps,min_delta=sys.maxsize)
+		_, num_steps, _ = model.train(n_steps,min_delta=sys.maxsize,patience=1)
```

Comment:
why is patience not the default here? Are you testing for it? If so, can you add a comment why you are setting it to 1?